frontend/src/main/java/org/abs_models/frontend/parser/XtextToJastAdd.java

```diff
@@ -1446,6 +1583,9 @@ private static ProductDecl fromXtext(final org.abs_models.xtext.abs.ProductDecla
 
     private static ProductExpr fromXtext(final org.abs_models.xtext.abs.ProductExpression xtext_exp) {
         ProductExpr result = null;
+        if (xtext_exp == null) {
```

Comment:
I believe the caller should check for null, and not call `fromXtext` in that case